docs/pt/docs/tutorial/dependencies/global-dependencies.md

```diff
@@ -0,0 +1,34 @@
+# Dependências Globais
+
+para alguns tipos de aplicação específicos você pode querer adicionar dependências para toda a aplicação.
```

Comment:
the capital letter is missing at the beginning

Reply:
Thanks for noticing :)